app/css/main.css

```diff
@@ -18,6 +18,45 @@ h3 {
   font-weight: normal;
 }
 
+h1.title {
+  font-size: 40px;
+  line-height: 48px;
```

Comment:
Let's remove `font-size: 40px` and `line-height: 48px` since they are already set for `h1` above.

Reply:
Thanks for the catch.